client/analysisui/layoutsupplierview.js

```diff
@@ -38,14 +38,16 @@ const LayoutSupplierView = function(params) {
         }
 
         this._items = newItems;
-        this.trigger("value_changed");
+
 
         if (_.isUndefined(this.supplierGrid) === false) {
             this.supplierGrid.suspendLayout();
             this.renderItemList();
             this.filterSuppliersList();
             this.supplierGrid.resumeLayout();
         }
+
+        this.trigger("value_changed");
```

Comment:
quotes ... and we don't use underscores in js ... (i don't even think we use them with constants in js (not sure about that one))

Reply:
i can change the quotes but the underscore is to big of a change. To many dependents in the dynamic property event system to change just like that.

Reply:
sure